src/main/java/Model/Database/Interaction/I_CommType.java

```diff
@@ -71,7 +76,7 @@ public static E_CommType retrieve(Connection conn, PreparedStatement ps, ResultS
     }
 
     public static E_CommType retrieveByName(Connection conn, PreparedStatement ps, ResultSet rs, String name) throws SQLException {
-        Assurance.IsVarcharOk(name);
+        Assurance.isVarcharOk(name);
```

Comment:
toto tu je opravene? s tym ze metoda isVarcharOk len vrati hodnotu true alebo false, ved tu nie je logika na checkovanie za tym ci?

Reply:
You are right. I will doplnit